content/management/limits/index.mdx

```diff
@@ -0,0 +1,52 @@
+---
+weight: 40
+category: "management"
+title: "Limits"
+tagline: "Resource limits in LiveChat Platform via dedicated APIs"
+desc: "There are some limitations to amount of different resources that can be created, based on license plan."
+---
+
+# Introduction
+
+Based on the <a href="https://www.livechat.com/pricing/" target="_blank">billing plan</a>, the LiveChat Platform limits the number of resources that can be created.
+
+# Resource Limits
+
+| Limited resource              | Starter | Team | Business | Enterprise |
+|-------------------------------|---------|------|----------|------------|
+| Agents                        | 400     | 400  | 400      | 400        |
+| Chat archives history         | 2m      | 1y   | 2y       | unlimited  |
+| Chat queue                    | 10      | 50   | 100      | 100        |
+| Chat queue (continuous chats) | 150     | 150  | 150      | 150        |
+| Groups                        | 1       | 20   | 40       | 80         |
+| Groups assigned to an agent   | 1       | 10   | 20       | 40         |
+| Groups in group chooser       | 1       | 25   | 25       | 25         |
+| Number of banned visitors     | 50      | 200  | 400      | 400        |
+| Tracked visitors              | 400     | 400  | 400      | 400        |
+
+There is a new type of error, `limit_reached`. Our APIs will return it when you try to create a resource over the limit. You may get this error when working with the following methods:
+
+## Agent Chat API
+
+- [List Archives](../../messaging/agent-chat-api/#list-archives)
+- [Ban Customer](../../messaging/agent-chat-api/#ban-customer)
+
+## Configuration API
+
+### v3
+
+- [Create Group](../configuration-api/#create-group)
+- [Update Group](../configuration-api/#update-group) *
+- [Create Agent](../configuration-api/#create-agent)
+- [Update Agent](../configuration-api/#update-agent) *
+
+* Note: this method can change groups assigned to an agent.
```

Comment:
```suggestion
**Note:** This method can change what groups an agent is assigned to.
```

Reply:
usually, you **assign sth (a task) to someone**, but in this context, you **assign users (agent) to the group**